css/theme-weebly.scss

```diff
@@ -908,6 +908,23 @@ div.Notice {
 	margin-right: 4px;
 }
 
+.tags .tags-link {
+	&:after {
+		content: ",";
+		cursor: default;
+		color: #000;
+		margin-right: 2px;
+	}
+
+	&:only-child,
+	&:first-child,
```

Comment:
Don't we want the comma after the first-child?

Reply:
Fixed.